src/Tests/Sagas/When_storing_saga_with_non_primitive_values.cs

```diff
@@ -54,7 +53,7 @@ static async Task<DictionaryTableEntity> GetEntity(Guid sagaId)
 
             try
             {
-                var tableEntity = (await table.ExecuteQueryAsync(query).ConfigureAwait(false)).SafeFirstOrDefault();
+                var tableEntity = table.ExecuteQuery(query).SafeFirstOrDefault();
```

Comment:
There's no async equivalent?

Reply:
Only the query segment version. I can easily switch to that but it felt a bit overkill for the tests to me

Reply:
👍